modules/adxcgAnalyticsAdapter.js

```diff
@@ -24,7 +24,7 @@ var adxcgAnalyticsAdapter = Object.assign(adapter(
   track({eventType, args}) {
     if (typeof args !== 'undefined') {
       if (eventType === CONSTANTS.EVENTS.BID_TIMEOUT) {
-        events.bidTimeout = [...new Set(args.map(item => item.bidder))];
+        events.bidTimeout = args.map(item => item.bidder);
```

Comment:
`Set` stores unique values, so I think this needs something like `filter(utilis.uniques)` after the `map`